spec/bolt/transport/winrm_spec.rb

```diff
@@ -130,25 +131,27 @@ def stub_winrm_to_raise(klass, message)
   end
 
   context "connecting over SSL", winrm: true do
+    let(:target) { make_target(host, ssl_port, ssl_config) }
```

Comment:
Turned out this was no longer testing SSL because config passed to transports was unused. Corrected by passing an SSL config when constructing the Target.